frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ResourceIcon } from './resource-icon';
+import {
+  K8sResourceCommon,
+  K8sResourceKind,
+  referenceForGroupVersionKind,
+  referenceForModel,
+  referenceForOwnerRef,
+  OwnerReference,
+} from '../../module/k8s';
+import { RootState } from '@console/internal/redux';
+import { withFallback } from '@console/shared/src/components/error/error-boundary';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
+import { k8sList } from '../../module/k8s/resource';
+
+const managedByLinkStateToProps = (state: RootState): managedByLinkStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+  };
+};
+
+const isOwnedByOperator = (csv: K8sResourceKind, owner: OwnerReference) => {
+  return (
+    csv.spec.customresourcedefinitions.owned.find((api) => api.kind === owner.kind) ===
+    csv.spec.customresourcedefinitions.owned.find((api) => owner.apiVersion === api.version)
```

Comment:
I might be misreading the code, but this still looks strange to me. Should it be one `find` call?

```suggestion
    !!csv.spec.customresourcedefinitions.owned.find((api) => api.kind === owner.kind && api.apiVersion === owner.apiVersion));
```

Reply:
Actually you probably want `Array.some` here instead of `Array.find`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some